ServiceClientVersions.json

```diff
@@ -0,0 +1,4 @@
+{
+  "sdk": "0.6.9",
+  "s3": "3.2.1"
+}
```

Comment:
This JSON file holds versions for individual service clients.  The `sdk` version is used for packages that aren't explicitly named.